docs_src/microservices/device/supported/device-usb-camera/supplementary-info/USB-protocol.md

```diff
@@ -0,0 +1,8 @@
+# USB Camera Device Service Specifications
+
+## USB Protocol Properties
+| Property | Description | EdgeX Value Type |               
+| -- | -- | -- |  
+| **Paths** | A list of internal [/dev/video paths](https://www.kernel.org/doc/html/v4.9/media/uapi/v4l/dev-capture.html) for the camera device. This list includes all streaming capable video paths for each device. | Object |  
```

Comment:
Right now the device example yml files have `Path` which needs to be changed to `Paths` right?

Reply:
Yeah, I'll add that to the tech debt story